backend/src/schema/resolvers/rooms.ts

```diff
@@ -25,6 +25,27 @@ export default {
       }
       return resolved
     },
+    UnreadRooms: async (object, params, context, resolveInfo) => {
+      const {
+        user: { id: currentUserId },
+      } = context
+      const session = context.driver.session()
+      const readTxResultPromise = session.readTransaction(async (transaction) => {
+        const unreadRoomsCypher = `
+          MATCH (:User { id: $currentUserId })-[:CHATS_IN]->(room:Room)<-[:INSIDE]-(message:Message)<-[:CREATED]-(user:User)
+          WHERE NOT user.id = $currentUserId AND NOT message.seen
+          RETURN toString(COUNT(DISTINCT room)) AS count
```

Comment:
```suggestion
          MATCH (:User { id: $currentUserId })-[:CHATS_IN]->(room:Room)<-[:INSIDE]-(message:Message)<-[:CREATED]-(sender:User)
          WHERE NOT sender.id = $currentUserId AND NOT message.seen
          RETURN toString(COUNT(DISTINCT room)) AS count
```
Just ignore this if you disagree - its correct

Reply:
I do not understand why you make `toString`

Reply:
I figured it aut <3